src/spacel/model/app.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, orbit, params=None):
         self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = params.get('loadbalancer', 'true')
+        self.availability = params.get('availability', 'private')
```

Comment:
Does this duplicate the `scheme` element? `internet-facing`/`internal` smells like `public`/`private`?
Or are we trying to split the ELB availability flag from the instance availability flag? if so: maybe rename (both?) to highlight that and consider consistent values?

When thinking naming, the fuzziest case is a `private multi-region` service:
You can declare an `internet-facing` service, but configure `sources` such that only private instances of the same orbit can connect. Your service will feel `scheme=internal`, but you need to set `internet-facing` for it to work across regions ( https://github.com/pebble/spacel-provision/blob/a17cc9f95ebf4d0a0376e69b75a0cdeb6fed5f46/src/spacel/provision/template/ingress_resource.py#L70 ). No clue how to make that intuitive :/